src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setColor(color);
```

Comment:
it's a bad practice to call methods inside of a constructor, for initialization of color field use the constructor of Figure, the same for other figures